src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs

```diff
@@ -65,5 +66,105 @@ public void ExpandEnvironmentVariables()
                 Environment.SetEnvironmentVariable(env, null);
             }
         }
+
+        [Benchmark]
+        public void GetEnvironmentVariables()
+        {
+            PerfUtils utils = new PerfUtils();
+            string env = utils.CreateString(15);
+            try
+            {
+                // setup the environment variable so we can read it
+                Environment.SetEnvironmentVariable(env, "value");
+
```

Comment:
Yeah, that is a good point.  We do drop the first iteration when we collect.  It is probably fine to not have warm-up iterations.